book/05-distributed-git/sections/contributing.asc

```diff
@@ -156,7 +156,7 @@ The basic format is `<oldref>..<newref> fromref -> toref`, where `oldref` means
 You'll see similar output like this below in the discussions, so having a basic idea of the meaning will help in understanding the various states of the repositories.
 More details are available in the documentation for https://git-scm.com/docs/git-push[git-push].
 
-Continuing with this example, shortly afterwards, John makes some changes, commits them to his local repository, and tries to push them to the same server:
+Continuing with this example, shortly afterwards, John tries to push them to the same server:
```

Comment:
Let's reword this sentence thus:

> Continuing with our example, John then tries to push his local changes to the same server:

Reply:
Well, however myself wonders why phrase ", shortly afterwards, John ... tries" was able to work before change in PR but it isn't with PR.

Reply:
From current point of view I am neither able nor ready to defend this proposal if one day it should need that.

Reply:
OK, let's stick to just your suggestions then. This isn't quite grammatically correct, let's change it to this:

> Continuing with this example, shortly afterwards, John tries to push his changes to the same server: